tests/sets/set_physical_devices_tests.yaml

```diff
@@ -0,0 +1,7 @@
+SetPhysicalDevicesTests:
+  Cases:
+    - Case: TestSeniorAppPersonalCarePartnerMessagePromptAndCallBackButton
+    - Case: TestSeniorAppAnswerCallForProviderCalling
+    - Case: TestCallPersonalCareButtonOnFullSchedule
```

Comment:
Is this test from care partner or senior?. Let's add the distinction to the title